src/components/base.scss

```diff
@@ -5,6 +5,8 @@
 .be {
     @include reset;
```

Comment:
What's the difference between this reset and the one below?

Reply:
@pramodsum Looks like the`@include reset` is for clearer rendering and a couple of other things. The import from box-react-ui is for overriding browser defaults for padding, border-box, margin, etc.